### ISSUE TITLE
when running the example, Intervar gets "stuck"at reading annotation database humandb/hg19_rmsk.txt

### ISSUE THREAD

**Issue by XimenaBo** [WGLab/InterVar#29]
Hello,

I am a newbie trying to run Intervar in an Amazon Linux AMI. I downloaded and installed ANNOVAR and then Intervar, and I've followed the instructions to run it. Everything goes well until it is time to read the rmsk file. It's been stuck at this point for a few hours and it seems excessive since I am just running the 18 SNP example "ex1.avinput".
Here is where it's at:

...
...
NOTICE: Scanning filter database humandb/hg19_dbnsfp31a_interpro.txt...Done
-----------------------------------------------------------------
NOTICE: Processing operation=r protocol=rmsk

NOTICE: Running with system command <annotate_variation.pl -regionanno -dbtype rmsk -buildver hg19 -outfile example/myanno example/ex1.avinput humandb>
NOTICE: Output file is written to example/myanno.hg19_rmsk
NOTICE: Reading annotation database humandb/hg19_rmsk.txt ...

What could be the problem? The humandb folder contains the right file, so it should be accessible.
Thank you in advance for your help

**Comment by kaichop** [comment]
you check memory usage and see if this is the problem (i.e. perhaps virtual
memory is used when reading this file due to lack of memory in your virtual
machine).

**Comment by XimenaBo** [comment]
Thank you Kai,
Yes, that is indeed the problem. There is apparently only 0.5 GiB available on the machine I'm using. I will upgrade to a tier with better resources. I know that regarding memory, more is always better, but what is the minimum required to run Intervar in a reasonable amount of time, for a file of the size of the example provided with the installation? I want to see if Intervar is what I need before going full into amazon cloud services.
Thank you!

**Comment by kaichop** [comment]
For the small sample, you are better off just running in a typical laptop
(use a virtual machine, or simply install windows version of Perl, or use a
Mac, or use a Linux installation) which should have at least 4G memory
nowadays. There is no point in renting a 500Mb virtual machine in amazon
for genome scale analysis.

**Comment by XimenaBo** [comment]
Yes, thank you. I just thought I would try it the free/cheap per hour amazon VM before fiddling around with my old and under-powered laptop. I guess I will just go ahead and do it. Thank you for the help! :)

**Comment by Aargol** [comment]
seems like file hg19_rmsk.txt has been removed from ANNOVAR available downloads?
how should i generate the file myself?
